util/generate-sample-ledger/src/lib.rs

```diff
@@ -131,23 +138,35 @@ pub fn bootstrap_ledger(
     }
 
     for (block, block_contents) in blocks_and_contents {
-        db.append_block(&block, &block_contents, None).unwrap();
+        db.append_block(&block, &block_contents, None).expect("Duplicate key, because of re-using old sample ledger. Delete target/sample_data/ledger  to start from scratch.");
```

Comment:
Nit: Log the actual path
```suggestion
        db.append_block(&block, &block_contents, None).unwrap_or_else(|err| {
            panic!(
                "Duplicate key, probably because of re-using old sample ledger. Delete {:?} to start from scratch.\nError: {}",
                path,
                err
            )
        });
```
(needs formatting)